joints_home: [0, -90, 0, -90, 0, -90]
joints_fold: [0, -175, 165, -180, 90, 90]
joints_routine: [ 
  [0, -70, -80, -120, 90, -80]
]